### DISCUSSION TITLE
How to use monai for image  normalization to zero-mean and unit-variance？

### DISCUSSION THREAD

**Discussion by AustinYuAo** [Project-MONAI/MONAI#6174] [Q&A] [answered]
Hi: 

How to use MONAI for image normalization to zero-mean and unit-variance?

**Comment by KumoLiu** [comment, marked as answer]
Hi @aubest, you could use `NormalizeIntensity`.
https://github.com/Project-MONAI/MONAI/blob/baa17a8b32c933fe00ae42a5f1990fdc0ad8f401/monai/transforms/intensity/array.py#L610-L617

Hope it can help you, thanks!
